util/top_gen/generator.py

```diff
@@ -364,8 +344,8 @@ def legacy_ordering(input_pin: OutputBlockIo) -> tuple[int, int, int]:
         ("rtl/templates/pinmux.sv.tpl", "rtl/system/pinmux.sv"),
         ("doc/ip/pinmux.md.tpl", "doc/ip/pinmux.md"),
     ):
-        print("Generating from template: " + template_file)
+        print("Generating from template:", template_file)
```

Comment:
Is this removal of the space after the colon intentional?

Reply:
Yep, the default  value for python `print`'s `sep` argument is a space.

Reply:
Ok, great.